### ISSUE TITLE
Missing topic and offset infos when login error in ConsumerOffsetCommitter

### ISSUE THREAD

**Issue by bmaggi** [confluentinc/parallel-consumer#629]
First thanks for this library, 

I'm trying to use it at its best to get the maximum throughput in my application. 

I got some issue when scaling up and down my app, due to "infinite rebalance". 

I feel that something may be going wrong between these two configurations : PERIODIC_CONSUMER_ASYNCHRONOUS and "partition.assignment.strategy" = org.apache.kafka.clients.consumer.CooperativeStickyAssignor 

I would be very helpful to add topic/partition/offset that are tried to be commited in this log (Ideally the ones that failed)

https://github.com/confluentinc/parallel-consumer/blob/master/parallel-consumer-core/src/main/java/io/confluent/parallelconsumer/internal/ConsumerOffsetCommitter.java#L100

I don't see these informations in the current message 
```
{extendedStackTrace=org.apache.kafka.common.errors.RebalanceInProgressException: Offset commit cannot be completed since the consumer is undergoing a rebalance for auto partition assignment. You can try completing the rebalance by calling poll() and then retry the operation.
, localizedMessage=Offset commit cannot be completed since the consumer is undergoing a rebalance for auto partition assignment. You can try completing the rebalance by calling poll() and then retry the operation., name=org.apache.kafka.common.errors.RebalanceInProgressException, commonElementCount=0, message=Offset commit cannot be completed since the consumer is undergoing a rebalance for auto partition assignment. You can try completing the rebalance by calling poll() and then retry the operation.}
``` 

Regards,
Benoit

**Comment by OmShinde1513** [comment]
Hi @rkolesnev , Can I work on this issue ?

**Comment by rkolesnev** [comment]
Hi @OmShinde1513 - sure.

**Comment by astubbs** [comment]
Tracked in the maintained fork as astubbs/parallel-consumer#168, which carries a summary of this
issue and its current status there.

The **Confluent-hosted version** of this project is no longer maintained - its README says so.
**Parallel Consumer itself continues** in my fork: https://github.com/astubbs/parallel-consumer

Nothing in this repository is closed or changed by this comment.

**Comment by astubbs** [comment]
@bmaggi - Fixed and released in the maintained fork, in [v0.6.0.0](https://github.com/astubbs/parallel-consumer/releases/tag/v0.6.0.0). The coordinates changed with the fork - on Maven Central it is now:

```xml
<dependency>
    <groupId>bz.stub.parallelconsumer</groupId>
    <artifactId>parallel-consumer-core</artifactId>
    <version>0.6.0.0</version>
</dependency>
```

(`parallel-consumer-vertx`, `parallel-consumer-reactor` and `parallel-consumer-mutiny` likewise.) The Java package is `bz.stub.parallelconsumer` too, so imports change with it - one command rewrites them (GNU `sed`; on macOS use `sed -i ''`):

```bash
find . -name '*.java' -exec sed -i 's/io\.confluent\.parallelconsumer/bz.stub.parallelconsumer/g' {} +
```

The short version of the fix: the failed-commit log line names every topic, partition and offset in the commit, with each entry's offset-map metadata reduced to its length so the line survives log truncation, and the full map one level down at `DEBUG`. The full account is on the fork's mirror, astubbs/parallel-consumer#168.
